data/jibauni/jibauni missions.txt

```diff
@@ -0,0 +1,579 @@
+		random < 45
+		not "event: jibauni rebel jibauni peace"
+	on offer
+		conversation
+			`As you are walking through the spaceport, you collide with a younger Jibauni. He immediately stands up and apologizes, then notices who you are. "Oh, the strange alien! Hello! May I ask of you a favor?"`
+			choice
+				`	"I guess."`
+				`	"Sorry for bumping into you. Sure you can ask!"`
+				`	"Nope, not right now. Other things to do."`
+					goto decline
+			`	"Would you be willing to transport me, Dran-lin, to the planet of <planet>? I would be interested in seeing the Rebel Jibauni and how they operate. I may end up joining them, or else I will ask you to take me back to a Jibauni planet."`
+			choice
+				`	"You know, the Rebel Jibauni are bad creatures. Are you sure you want to go?"`
+					goto rebelsbad
+				`	"The wonderful Rebel Jibauni? Sure, I'll take you immediately!"`
+					goto rebelsgood
+			label rebelsbad
+			action
+				set "jibauni: dran-lin: says rebels are bad"
+			`	"I would like to at least check it out. Come on, please take me?"`
+				goto choice
+			label rebelsgood
+			action
+				set "jibauni: dran-lin: says rebels are good"
+			`	"I am... surprised that you say that. Most I have met give them a bad reputation and a good scolding. But I would be happy to be taken!"`
+			label choice
+			choice
+				`	"Alright, I'll take you."`
+					accept
+				`	"Anyway, I need to do something else, so I'm not going to take you after all."`
+			label decline
+			`	The Jibauni drifts off disappointedly.`
+				decline
+	on complete
+		payment
+		event "jibauni: dran-lin gone" 1
+		conversation
+			`Dran-lin says, "I am going to explore the spaceport. Could you meet me at the main hotel in a few hours? If you do not, I will assume you do not want to take me back."`
+
+event "jibauni: dran-lin gone"
+
+mission "Jibauni: Dran-lin 1"
+	priority
+	name "Take Dran-lin to <planet>"
+	description "Dran-lin didn't love the way <origin> felt, and would like you to take him to <destination> to see if it was just <origin> and not all Rebel planets that were like that."
+	passengers 1
+	source
+		government "Jibauni (Rebel)"
+	destination
+		government "Jibauni (Rebel)"
+		distance 1 1
+	to offer
+		has "Jibauni: Dran-lin 0: done"
+		not "event: jibauni: dran-lin gone"
+	on offer
+		conversation
+			`Dran-lin meets you at the main hotel, which happens to be called Jivam Dar-li. "Could you take me to <destination>? I do not get a good sense from the spaceport of <origin>. Perhaps this city in particular is just bad."`
+			choice
+				`	"Alright."`
+				`	"Nope, sorry. I agreed to take you... once. No more."`
+					goto declining?
+			`	"Perfect! I will be ready when you are."`
+				accept
+			label declining?
+			`	"You can not do that! It is not very kind; there is unlikely to be anybody who will take me back to Jibauni space. Please take me!"`
+			choice
+				`	"Oh, whatever. I'll take you to <planet>."`
+					accept
+				`	"My word is my word. Goodbye."`
```

Comment:
```suggestion
				`	"I meant what I said. Goodbye."`
```
just feels like you're kind of breaking the unstated promise to bring them back - so feels a little awkward to say your word is your word.